index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="icon" href="images/header-background-mobile.png" type="image/x-icon"/>
+    <title>Supreeti Portfolio</title>
+  </head>
+  <body>
+    <header class="header1">
+        <h3 class="logo"><a href="#">My Logo</a></h3>
+          <nav>
+              <img src="images/Union.png" alt=""/>
+          </nav>
+    </header>
+    <main>
+        <section class="section">
+            <h1 class="title">I’m Supreeti Glad to see you!</h1>
+            <p>
+              I’m a software developer I can help you build a product, feature
+              or website Look through some of my work and experience! If you like
+              what you see and have a project you need coded, don’t hestiate to
+              contact me.
+            </p>
```

Comment:
- [ ] You are yet to apply the font styles from the Figma design and that could be a reason why can't apply the correct font sizes to these texts. Please apply the fonts specified in the Figma design. You can easily download the fonts from [Google Fonts](https://fonts.google.com)